osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -304,6 +321,15 @@ private void refresh()
 
             difficultyIconContainer.FadeInFromZero(500, Easing.OutQuint);
             mainFillFlow.FadeInFromZero(500, Easing.OutQuint);
+
+            downloadTracker?.RemoveAndDisposeImmediately();
+
+            if (beatmap != null)
+            {
+                Debug.Assert(beatmap.BeatmapSet != null);
+                downloadTracker = new BeatmapDownloadTracker(beatmap.BeatmapSet);
+                AddInternal(downloadTracker);
```

Comment:
This is too much overhead just to maybe display context menu items at some point.

I dislike download trackers being everywhere and don't want ones like this that are probably never even queried. Just use a `BeatmapManager` query at the point of constructing the menu items.

Reply:
Had considered querying `BeatmapManager`, but thought it wouldn't hurt to add a download tracker (well, maybe bad because there's literally a download button next to the tracker, but yeah).